.circleci/config.yml

```diff
@@ -3,6 +3,7 @@ version: 2.1
 orbs:
   frontend-tools: contentful/frontend-tools@1
   vault: contentful/vault@1
+  nx: nrwl/nx@1.6.1
```

Comment:
let's remove this orb for now. It would only show any effect if you would call all the npm commands in this file through `nx affected` which is currently not working with our CircleCI setup. We are working on a fix and will eventually provide a PR for all repositories.